.copr/Makefile

```diff
@@ -17,6 +17,12 @@ srpm:
 		"bdcs-cli.spec") CLONE_URL="https://github.com/weldr/bdcs-cli.git" ; \
 		                 CABAL_PATH="BDCSCli.cabal" ; \
 				 CABAL_SPEC_OPTIONS="-b" ;; \
+		"codec-rpm.spec") CLONE_URL="https://github.com/weldr/codec-rpm.git" ; \
+		                  CABAL_PATH="codec-rpm.cabal" ; \
+		                  CABAL_SPEC_OPTIONS="-b" ;; \
```

Comment:
These shouldn't need the `-b` option to cabal-rpm, since they only provide libraries. CABAL_SPEC_OPTIONS should be empty for both.